hack/ostests/.terraform.lock.hcl

```diff
@@ -1,110 +1,125 @@
-# This file is maintained automatically by "tofu init".
+# This file is maintained automatically by "terraform init".
```

Comment:
hmm we're geting into toolwars here ... CI uses OpenTofu ...

Reply:
Entirely removed that commit of the PR, it's not required anyway.